### ISSUE TITLE
generate an issues fixed list based on all the dependent projects?

### ISSUE THREAD

**Issue by jstrachan** [fabric8io/fabric8-platform#94]
it'd be great to be able to generate a list of all the issues that have been fixed by an individual fabric8-platform release (based on all the issues from kube model/client, fabric8, devops, ipaas, quickstarts, fabric8-console, gofabric8)

I wonder if we can make some tool to query all the issues fixed in a bunch of repos and consolidate a HTML report?

**Comment by jstrachan** [comment]
maybe we run a changelog generator on each individual project (kubernetes-client, fabric8, fmp, devops et al) - then have a composition tool that grabs the changelog for each dependency and collates them?

**Comment by rawlingsj** [comment]
Not sure if it helps but I had a play with the golang github code which works nicely https://github.com/fabric8io/goimprove/blob/master/cmds/ghDownloads.go#L73

Might be a quick and easy way to get the data you want.  Also easily send events to elasticsearch..

**Comment by jstrachan** [comment]
yeah - was hoping we could generate the changelog of fixed issues per release; put it into some canonical place - then could aggregate the data for the platform releases (fabric8-platform, ipaas-platform)
